hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/HoodieSparkUtils.scala

```diff
@@ -199,6 +199,31 @@ object HoodieSparkUtils extends SparkAdapterSupport with SparkVersionsSupport wi
     }
   }
 
+  def getJsonRdd(df: DataFrame): RDD[String] = {
+    df.toJSON.rdd
```

Comment:
Should we handle empty df?

Reply:
Maybe, but that method isn't needed so I'm just gonna delete it